pyproject.toml

```diff
@@ -1,14 +1,17 @@
 [build-system]
-requires = ["hatchling"]
-build-backend = "hatchling.build"
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
 
-[project]
+[tool.poetry]
 name = "mavedb"
-dynamic = ["version"]
+version = "2023.5.0"
```

Comment:
is there a reason we can't do `dynamic` here? in any case, you probably want `2024.0.0` here (or `.1.0` since that's the branch this is headed for). you'll probably want to edit the wiki to reflect that this is an additional place the version number must be changed when a release is cut.

Reply:
I think we may be able to just place the version here in the `pyproject.toml` file instead of in the `__version__` file and only have to worry about the one here. It seems more natural to me to have the version in the project file anyways.

In any case I'll update it to reflect the intended destination.

Reply:
Hmm actually I guess that would make it tough to surface the version via the API endpoint. I'll investigate whether this is possible with Poetry.

Reply:
Poetry itself just duplicates the version between a `__version__` file and the `pyproject.toml` file, which I know is kinda annoying. 

It seems like [the way people get around this](https://github.com/python-poetry/poetry/issues/273#issuecomment-570999678) is by using `importlib.metadata.version('package-name')` to get the version of a package. We don't currently install the project in the pip/Poetry sense, so wouldn't be able to use that method at the moment. If we wanted to install it we could, but it would be redundant with copying the source code in the Docker file, and the only reason we'd be doing it would be to get the version string. So I guess I'm inclined to just duplicate the version.